distribution/com.modrinth.ModrinthApp.yml

```diff
@@ -0,0 +1,88 @@
+id: com.modrinth.ModrinthApp
+runtime: org.gnome.Platform
+runtime-version: "42"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+  - org.freedesktop.Sdk.Extension.node18
+
+command: modrinth-app
+finish-args: 
+  - --filesystem=xdg-run/app/com.discordapp.Discord:create # allows for discord rpc
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+  - --socket=wayland
+
+build-options:
+  append-path: "/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/node18/bin"
+
+  build-args:
+    - --share=network
+
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+
+  # for some controller mods (see https://github.com/isXander/Controlify/issues/31)
+  - shared-modules/libusb/libusb.json
+
+  - name: flite
+    buildsystem: autotools
+    config-opts:
+      - --enable-shared
+      - --with-audio=pulseaudio
+    sources:
+      - type: archive
+        url: https://github.com/festvox/flite/archive/refs/tags/v2.2.tar.gz
+        sha512: 1ca2f4145651490ef8405fdb830a3b42e885020a7603d965f6a5581b01bed41047d396b38c2ceab138fc0b28d28078db17acd2b5a84c6444cb99d65c581afa72
+    cleanup:
+      - '/lib/libflite*.a'
+
+  - name: modrinth-app
+    buildsystem: simple
+    build-options:
+      append-path: /run/build/modrinth-app/pnpm
+    build-commands:
+      - pnpm config set store-dir .pnpm-store
+      - cd theseus/theseus_gui && pnpm install
+      - cd theseus/theseus_gui && pnpm build
+      - cd theseus && cargo --offline fetch --manifest-path Cargo.toml
+      - cd theseus && cargo --offline build --release
```

Comment:
No need to build offline, network access is already allowed

Reply:
i know, i just thought it would still be nice to only use networking when needed

plus it makes it a bit easier for when later when network access won't be allowed :p